etc/kayobe/kolla.yml

```diff
@@ -150,6 +150,10 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/octavia.git
     reference: stackhpc/{{ openstack_release }}
+  blazar-base:
+    type: git
+    location: https://github.com/stackhpc/blazar
+    reference: stackhpc/master
```

Comment:
If we're pulling this in we should probably rebuild and repin the Blazar container so it doesn't catch anyone out too badly. I've started a build [here](https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/12007091927)

Reply:
New tag is 20241125T093138